commitlint.config.js

```diff
@@ -1 +1,3 @@
-module.exports = { extends: ['@commitlint/config-conventional'] };
+module.exports = {
+  extends: ['./common/autoinstallers/commitlint/node_modules/@commitlint/config-conventional/index.js']
```

Comment:
why was the path change necessary?

Reply:
commitlint is not working well with pnpmV7 and this is a workaround : https://github.com/conventional-changelog/commitlint/issues/3397